packages/office-ui-fabric-react/src/components/List/examples/List.Ghosting.Example.tsx

```diff
@@ -70,38 +55,32 @@ const classNames: IListGhostingExampleClassObject = mergeStyleSets({
   },
 });
 
-export class ListGhostingExample extends React.Component<IListGhostingExampleProps> {
-  private _items: IExampleItem[];
-  constructor(props: IListGhostingExampleProps) {
-    super(props);
-    this._items = props.items || createListItems(5000);
-  }
+const onRenderCell = (item: IExampleItem, index: number, isScrolling: boolean): JSX.Element => {
+  return (
+    <div className={classNames.itemCell} data-is-focusable={true}>
+      <Image
+        className={classNames.itemImage}
+        src={isScrolling ? undefined : item.thumbnail}
+        width={50}
+        height={50}
+        imageFit={ImageFit.cover}
+      />
+      <div className={classNames.itemContent}>
+        <div className={classNames.itemName}>{item.name}</div>
+        <div className={classNames.itemIndex}>{`Item ${index}`}</div>
+      </div>
+    </div>
+  );
+};
 
-  public render(): JSX.Element {
-    return (
-      <FocusZone direction={FocusZoneDirection.vertical}>
-        <div className={classNames.container} data-is-scrollable={true}>
-          <List items={this._items} onRenderCell={this._onRenderCell} />
-        </div>
-      </FocusZone>
-    );
-  }
+export const ListGhostingExample: React.FunctionComponent = () => {
+  const [items] = React.useState(() => createListItems(5000));
```

Comment:
`useConst(() => createListItems(5000));`

Same in other list examples that currently use the `useState` pattern for item creation.